app/matplotlib.ipynb

```diff
@@ -233,22 +230,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
-   "outputs": [],
    "source": [
     "x = np.linspace(0, 5, 100)\n",
     "y1 = np.sin(x)\n",
     "y2 = np.cos(x)\n",
     "\n",
-    "# write your code here\n"
+    "fig, axs = plt.subplots(2)\n",
+    "fig.suptitle(\"Sin and Cos Functions\")\n",
+    "axs[0].plot(x, y1)\n",
+    "axs[0].set_title(\"Sin Function\")\n",
+    "axs[0].set_xlabel(\"x values\")\n",
+    "axs[0].set_ylabel(\"sin(x)\")\n",
+    "\n",
+    "axs[1].plot(x, y2)\n",
+    "axs[1].set_title(\"Cos Function\")\n",
+    "axs[1].set_xlabel(\"x values\")\n",
+    "axs[1].set_ylabel(\"cos(x)\")\n",
+    "\n",
+    "plt.show()\n"
```

Comment:
In Task 6, the subplots should be displayed horizontally. You can achieve this by setting the `nrows` parameter to 1 and `ncols` to 2 in the `plt.subplots()` function.